deployer/template/aws_elasticloadbalancingv2_loadbalancer.go

```diff
@@ -0,0 +1,96 @@
+package template
+
+import (
+	"fmt"
+
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/awslabs/goformation/cloudformation/resources"
+	"github.com/coinbase/fenrir/aws"
+	"github.com/coinbase/fenrir/aws/sg"
+	"github.com/coinbase/fenrir/aws/subnet"
+)
+
+func ValidateAWSElasticLoadBalancingV2LoadBalancer(
+	projectName, configName, resourceName string,
+	template *cloudformation.Template,
+	ec2c aws.EC2API,
+	res *resources.AWSElasticLoadBalancingV2LoadBalancer,
+) error {
+	res.Name = normalizeName("fenrir", projectName, configName, resourceName, 32)
```

Comment:
Given the length requirement, and that we are including fenrir in the name, we will almost certainly have collisions here. 

I almost want to just hash all the components so we are guaranteed a unique, albeit random, name. Then we just use tags to find the ALB.

Reply:
normalizeName adds an 8-character hash to the end, which should be sufficient to avoid clashes